flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/sink/writer/AsyncSinkWriter.java

```diff
@@ -181,15 +187,88 @@ public abstract class AsyncSinkWriter<InputT, RequestEntryT extends Serializable
      * <p>During checkpointing, the sink needs to ensure that there are no outstanding in-flight
      * requests.
      *
+     * <p>This method is deprecated in favor of {@code submitRequestEntries( List<RequestEntryT>
+     * requestEntries, ResultHandler<RequestEntryT> resultHandler)}
```

Comment:
`<p>This method is deprecated in favor of {@code submitRequestEntries(List<RequestEntryT>
     * requestEntries, ResultHandler<RequestEntryT> resultHandler)}`